src/App.js

```diff
@@ -1,10 +1,28 @@
 import React, { Component } from 'react';
-import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
+import './App.scss';
+
 export class App extends Component {
-  state = {};
+  state = {
+    query: '',
+    isVisible: moviesFromServer,
+  }
+
+  changeQuery = (event) => {
+    this.setState({
+      query: event.target.value,
+      isVisible: moviesFromServer.filter(
```

Comment:
```suggestion
      visibleMovies: moviesFromServer.filter(
```

Reply:
Good point

Reply:
As a variant you also can filter the movies inside `render` method (not in the state)